README.md

```diff
@@ -2,26 +2,12 @@
 
 ## Required on your Machine:
 1. `git-secrets`: Install following instructions [here](https://github.com/awslabs/git-secrets)
-2. Yarn 2+ (see below)
+2. Yarn
 3. NodeJS 16.x
 
-## Installing Yarn >= 2
-
-- Open a terminal *OUTSIDE* of the project folder
-- Make sure node >=16.10.0 (at time of test 16.51.1 was the latest and worked)
-- Make sure yarn >=1.22 is installed
-- run "corepack enable"
-- run "yarn -v"
-- It should >=1.22
-###
-- Open a terminal *INSIDE* of the project folder
-- run "yarn set version stable"
-- run "yarn -v"
-- should be >= 2.0.0 (at time of test v3.2.1 was the ref. in this project)
-
 ## To Run
 
-- From the root run `yarn dev`
+- From the root run `yarn dev --filter=YOUR_PROJECT_NAME` (otherwise it will try to run all projects in dev mode)
```

Comment:
Is this not true still (this is a turborepo specific thing, rather than yarn?)

Reply:
The old readme suggestion was problematic in that running `yarn dev` from the root will always try to run every project in the monorepo at once. This suggestion should save a few headaches if people are new to turbo/monorepos :)

Reply:
Must have been my late-night brain, but for some reason I saw this as you removing the extra info from the README, rather than adding it 😆 

Forget I said anything… :disappears: